<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Randevu Al</string>
  </property>
  <widget class="QComboBox" name="cityBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Şehir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>İlçe</string>
   </property>
  </widget>
  <widget class="QComboBox" name="countyBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="clinicBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Klinik</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hospitalBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Hastane</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2099</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>8</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2099</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>8</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Başlangıç Tarihi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitiş Tarihi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Randevu Ara</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
